sdk/aqueduct/utils.py

```diff
@@ -197,6 +199,13 @@ def serialize_function(
         with open(os.path.join(dir_path, MODEL_PICKLE_FILE_NAME), "wb") as f:
             pickle.dump(func, f)
 
+        # Write function source code to file
+        source_file = "{}.py".format(op_name)
+        with open(os.path.join(dir_path, source_file), "w") as f:
+            source = inspect.getsource(func)
```

Comment:
Based on the loom vid, this won't capture the import statements right? Should that be a nice-to-have follow up (is it even possible)

Reply:
Yup that's right. I don't think it's easily possible and not sure if its worth the engineering effort cc @vsreekanti